documentation/homework/week1/todo-app/src/components/Todo-item.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react'
+
+export default class TodoItem extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
```

Comment:
This state probably needs to be lifted to your `App.js` component where you are loading the data from a json file in the first place. I would recommend loading that json file into `App.js`'s state, and then rendering out these `Todo-Item`s with only props and event handlers.

For instance, `App.js` has no way of knowing which todos are checked and which are not. What if you needed to display a total "checked todos" count?